src/js/discharge-wizard/containers/GuidancePage.jsx

```diff
@@ -16,6 +16,13 @@ class GuidancePage extends React.Component {
 
   componentDidMount() {
     window.scrollTo(0, 0);
+    localStorage.setItem('dw-viewed-guidance', true);
```

Comment:
all of the sessionStorage and localStorage stuff is for making sure we have desired behavior around people trying to navigate to certain pages directly, in the same window or in a new tab